Lib/inspect.py

```diff
@@ -537,7 +537,7 @@ def _getmembers(object, predicate, getter):
     processed = set()
     names = dir(object)
     if isclass(object):
-        mro = (object,) + getmro(object)
```

Comment:
MRO always contains the type so this redundant.

Reply:
Is this not covered by any test?

Reply:
This is covered in test. Adding object to the tuple of bases is redundant here and leads to object occurring twice in mro.